### ISSUE TITLE
Making prompt segments like ruby_version_prompt optional

### ISSUE THREAD

**Issue by davidpfarrell** [Bash-it/bash-it#1521]
Greetings All, 

TIL it appears that, in general, one cannot disable the `ruby_version_prompt`.

I recently reviewed #1520 which, in their first version of the PR, had completely removed the `ruby_versoin_prompt` segment from the theme.  I thought it was odd, but now I realize they likely did it because there was no other clear way to disable the prompt segment.

Up until now, I kinda assumed it only showed if the user was in a ruby source directory of some kind (which is how some of the other segments work), but it appears that, if the prompt is used, and if Ruby is installed, it will *always* show the ruby version in the prompt text.

I must say that sounds a bit annoying.

A [quick search](https://github.com/Bash-it/bash-it/search?p=2&q=ruby_version_prompt&unscoped_q=ruby_version_prompt) suggests this is accepted behavior across all themes that use the prompt.

In general, these themes don't `"build up"` their prompt on the idea of configurable segments (like say Powerline), so there are no theme-specific ways of excluding these types of segments.

Wondering if something like the following might be useful:

```
function ruby_version_prompt {
  if [ -z "${RUBY_VERSION_PROMPT_DISABLED}" ]; then
    echo -e "$(rbfu_version_prompt)$(rbenv_version_prompt)$(rvm_version_prompt)$(chruby_version_prompt)"
  fi
}
```

If we think this approach is useful, we could probably convince @shiguol to write up a PR since they were keen to disable it in their theme :)

-DF

---
cc: @shiguol

**Comment by nwinkler** [comment]
I like this idea - let's do this!

**Comment by davidpfarrell** [comment]
This was closed by #1520 - Nice work team !
